### ISSUE TITLE
Package Request: chromium-pepper-flash

### ISSUE THREAD

**Issue by crocket** [voidlinux/void-packages#13539]
adobe-flash-plugin is for firefox. chromium-pepper-flash is faster.

I request that the archive be hosted by void linux so that the download URL cannot be easily broken.

**Comment by Duncaen** [comment]
1. They stopped shipping pepper flash as shared library.
2. The license does not allow redistribution.

**Comment by Duncaen** [comment]
Maybe someone can package `adobe-flash-ppapi`.

**Comment by crocket** [comment]
There is already a pull request for adobe-flash-ppapi.

But, its download URL is broken.

**Comment by cr6git** [comment]
@crocket no, it's npapi.

**Comment by cr6git** [comment]
@crocket yes you're right I see [#7632](https://github.com/voidlinux/void-packages/pull/7632) now! sorry.
